File: MYCODES/PYTHON/basic programms/starpattern.py
# Q11. Write a C program to print the pattern:
# *
# **
# ***
# ****
# *****

# for i in range(1,5,1):
#     for j in range (1,i,1):
#         print("*")
#         print("\n")


for x in range(1,6):
    for y in range(1, x+1):
        print("*",end='')
    print()


# i=1
# j=1
# while (i<=5):
#     while (j<=i):
#         print("*")
#         j+=1
#         print("\n")
#         i+=i

File: MYCODES/JAVA/Loops in java/starpattern.java
// Q11. Write a C program to print the pattern:
// *
// **
// ***
// ****
// *****

package Loops;

import java.util.Scanner;

public class starpattern
{
    public static void main (String args [])
    {
        int i,j;
        for (i=1; i<=5; i++)
        {
            for (j=1; j<=i; j++)
            {
                System.out.print("*");
            }
            System.out.print("\n");
        }
    }
}
